BRLS/src/library/borrowitem/BorrowItemControl.java

```diff
@@ -100,17 +105,20 @@ public void borrowingCompleted() {
 
```

Comment:
This is missing closing brackets as well however the final changes show it. All the brackets look fine.